ILTransform/TestProject.cs

```diff
@@ -511,6 +512,14 @@ public static void GetKeyNameRootNameAndSuffix(string path, out string keyName,
             {
                 suffixIndex -= 2;
             }
+            else if (rootName.EndsWith("_TargetUnix"))
+            {
+                suffixIndex -= 11;
+            }
+            else if (rootName.EndsWith("_TargetWindows"))
+            {
+                suffixIndex -= 14;
+            }
```

Comment:
This brings up something that I don't know about our test system.  Do we build expect a particular build of src\test to be valid for all targets?  My guess would be that a particular build only supports one target - for example, there are lots of tests with C++ code that must be compiled to a specific architecture.

But that raises the question - why does this test have two project files instead of just one with an ifdef?

Reply:
In this case there was a constant definition that would be defined for Windows but not for Unix systems, that is the reason why there are two different csproj files, but you are right, we can just make that Conditional on the target and have only one csproj.

Reply:
I am reverting this commit change